README.md

```diff
@@ -37,6 +37,12 @@ The following items can be set via `--set` flag during installation or configure
 - **Disable**: The data does not survive the termination of a pod.
 - **Persistent Volume Claim(default)**: A default `StorageClass` is needed in the Kubernetes cluster to dynamic provision the volumes. Specify another StorageClass in the `storageClass` or set `existingClaim` if you have already existing persistent volumes to use.
 
+
+#### Configure the cluster security:
+
+- **Ldap**: Enable ldap to secure the cluster and add user/password authentication. when ldap is enabled make sure to change the variables `properties.isSecure` and `properties.clusterSecure` to **true** and set `properties.httpPort` to **null** and `properties.httpsPort` to **9443**.
+- Also to use ldap you need to set the **namesapce** and the **release-name** you're going to use in advance respectivly in `properties.namespace` and `properties.release` .
```

Comment:
The release name can be retrieved by .Release.Name so the user doesn't need to add it in values file.

Reply:
Why the namespace name is needed?

Reply:
Also I propose to add ldap in a sub-section as it's one of the possible auth method. You can also say that it's mandatory for nifi to enable the ssl to use an auth method.